firmware/brake/tests/features/step_definitions/receiving_messages.cpp

```diff
@@ -11,15 +11,17 @@ GIVEN("^the left brake sensor reads (.*)$")
 {
     REGEX_PARAM(int, sensor_val);
 
-    g_mock_arduino_analog_read_return = sensor_val;
+    g_mock_arduino_analog_read_return[14] = sensor_val;
+    g_mock_arduino_analog_read_return[14] = sensor_val;
```

Comment:
Is line 15 necessary? Does it do anything different compared to line 14?

Reply:
857217f